### PR TITLE
implement GitHub Actions CI

### PR DESCRIPTION
**Pull request by hirosassa**
## What

In this PR, I implement CI using GitHub Actions.

### CODE REVIEW

File: test/test_bigquery.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+import pytest
 
+from test.init import config, logger, ignore_warnings
 from src.lib.bigquery import Bigquery, ResultType
 from src.lib.dataset_desc import DatasetDesc
 from src.lib.table_desc import TableDesc
 
 GCP_PROJECT_ID = config.gcp_project
-TEST_DS = "test_bqdesc_buckuper"
+TEST_DS = "test_bqdesc_backuper"
```
Comment (hirosassa) [inline test/test_bigquery.py:12]: fix typo

File: tox.ini [hunk 2]
```diff
@@ -1,9 +1,18 @@
 [tox]
 envlist = py39, lint
 
+[pytest]
+testpaths = test
+addopts = -m "not gcp_project" --strict-markers
```
Comment (hirosassa) [inline tox.ini:6]: skip tests need GCP project access currently (I will implement mock test in next PR)

File: test/test_bigquery.py [hunk 3]
```diff
@@ -298,7 +300,7 @@ def test_update_table_not_exist_dataset(self):
         }
         table_desc = TableDesc(in_dict=new_table_dict)
         bq_update_result = self.bq.update_table_desc(table_desc)
-        self.assertEqual(ResultType.DATASET_NOT_FOUND, bq_update_result.type)
+        self.assertEqual(ResultType.TABLE_NOT_FOUND, bq_update_result.type)
```
Comment (hirosassa) [inline test/test_bigquery.py:303]: former test implementation is wrong, this should be `TABLE_NOT_FOUND`

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox tox-gh-actions
+    - name: Test and lint with tox
+      run: tox
```
Comment (hirosassa) [inline .github/workflows/test.yaml:26]: test workflow configurations

File: test/test_bigquery.py (hunk 1 above)
Comment (TetsutaroWatanabe) [reply test/test_bigquery.py:12]: thank you

File: test/test_bigquery.py (hunk 3 above)
Comment (TetsutaroWatanabe) [reply test/test_bigquery.py:303]: Thank you. It is bug.

I just realised that the function name `test_update_table_not_exist_dataset` is duplicated and the test doesn't work. 
Can you change the function name to something else?

File: tox.ini (hunk 2 above)
Comment (TetsutaroWatanabe) [reply tox.ini:6]: ok

File: test/test_bigquery.py (hunk 3 above)
Comment (hirosassa) [reply test/test_bigquery.py:303]: @TetsutaroWatanabe 
> the function name `test_update_table_not_exist_dataset` is duplicated

It looks not duplicated.

Comment (TetsutaroWatanabe) [reply test/test_bigquery.py:303]: I'm sorry my misunderstanding. 

In #7 you already renamed `test_update_table_not_exist_dataset` to `test_update_table_not_exist_table`

**Review by TetsutaroWatanabe — APPROVED** [review]
LGTM

**Pull request merged**